content/docs/dataform-web/guides/git-access-tokens.md

```diff
@@ -2,18 +2,48 @@
 title: Git access tokens
 ---
 
-Git access tokens are user to authenticate access to Dataform projects attached to remote git providers, such as GitHub.
-
-Acess tokens can be updated on the **project settings page**, within the app.
+Git access tokens are important for Dataform projects linked to remote git providers, such as GitHub. They provide authentication for accessing and controlling the git project.
 
 ## User authentication
 
-Access tokens for **users** must have both **read and write access** to your remote repository.
+User access tokens are used to authenticate git access of Dataform projects during project manipulation.
+
+Without a valid access token, users will be unable to do things such as create branches.
+
+They must be configured to have both **read and write access** to your remote repository.
+
+They can be added or removed on the **user settings page**.
+
+## Scheduler authentication
+
+Scheduler access tokens are used to authenticate git access of Dataform projects when running automated schedules.
+
+Without a valid access token, the scheduler will not be able to read the contents of the project and perform the run.
+
+They must have **read** access to your remote repository.
+
+They can be updated on the **project settings page**, within the app.
 
-Access tokens for the **scheduler** must have **read** access to your remote repository.
+For smaller projects (e.g. 5 or fewer people), it's ok to use a user's personal access token to authenticate the scheduler, but for larger teams a **machine user** is recommended. If you have a DevOps team, ask them about whether a machine user already exists.
+
+### Machine user
+
+machine users are generally regular user accounts, but used purely to run automated (or triggered) tasks.
+
+If a user's personal access token is used, and then they leave the organization or their permission is revoked, then the scheduler will fail.
+
+Because of this, **we recommend to have a global machine user on organization level, to make sure there is no disruption during scheduling**.
+
+Check [this](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users) for further info on creating machine users.
+
+Before making a machine user on GitHub, please read the [TOCs](https://help.github.com/en/github/site-policy/github-terms-of-service#3-account-requirements).
```

Comment:
```suggestion
Before creating a machine user on GitHub, please read the [TOCs] (https://help.github.com/en/github/site-policy/github-terms-of-service#3-account-requirements) from GitHub.
```

Reply:
`Before creating a machine user on GitHub, please read their [TOCs](https://help.github.com/en/github/site-policy/github-terms-of-service#3-account-requirements).`?